docs/bit-components/overview.md

```diff
@@ -0,0 +1,220 @@
+---
+id: component-overview
+title: Component Overview
+---
+
+A Bit component a set of files, configuration and dependencies, implementing a reusable and composable functionality, such as:
```

Comment:
This should probably say `A bit component is a set` (note addition of `is`).

I started to PR this myself, but then saw your PR changed this file already, so figured I'd note it here 😃